package/kubectl-shell.sh

```diff
@@ -15,11 +15,6 @@ if [ -z "${cluster}" ]; then
     exit 1
 fi
 
-if [ -z "${cacerts}" ]; then
```

Comment:
given we added this check to fix the issue around cert rotation, just wanted to make sure we are not breaking that previously added fix

Reply:
Yes, the original fix is still in the `setup-shell.sh` script. Here I just remove the input check. I tested the original scenario and several other paths and it passed.